src/test/java/uk/gov/hmcts/reform/bulkscan/orchestrator/SampleData.java

```diff
@@ -168,7 +168,7 @@ public static Envelope envelope(Classification classification, String jurisdicti
                     String.format("type_%s", index),
                     String.format("subtype_%s", index),
                     ZonedDateTime.parse("2018-10-01T00:00:00Z").plus(index, DAYS).toInstant(),
-                    String.format("https://example.gov.uk/uuid%s", index),
+                    String.format("https://example.gov.uk/files/uuid%s", index),
```

Comment:
files -> document ?

Reply:
doesn't matter but would be nice :)

Reply:
this is context-path which can be anything. Also, this class is used only in unit tests